project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -5,11 +5,11 @@
 To establish and administer a fund for members of the OpenJS Foundation to travel
 and spread knowledge about and support the Javascript ecosystem and the Foundation.
 
-### [Current Year Fund](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/TRAVEL_FUND/2022.md)
+### [Current Year Fund](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/TRAVEL_FUND/2023.md)
 
-### [2021 funds](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/TRAVEL_FUND/Achived_Travel_Fund_Records/2021.md)
+### [2022 Funds](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/TRAVEL_FUND/Achived_Travel_Fund_Records/2022.md)
 
-### [2019 funds](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/TRAVEL_FUND/Achived_Travel_Fund_Records/2019.md)
+### [2021 funds](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/TRAVEL_FUND/Achived_Travel_Fund_Records/2021.md)
```

Comment:
did we not have a 2020 fund?

seems like we should list all of them, just just the prior 2 years?

Reply:
There doesn't seem to be any sign of a 2020 fund, so I'm guessing there wasn't one.

Also I'm not sure if we should list all of them, I think we should keep listing the most relevant ones to date because there's a high chance most people don't look at the old ones, and that this list will keep getting bigger on every new fund entry.

Reply:
Why bother to list the two previous years then?

If 2021 is relevant than so is 2019 imo. If the list gets too big, that's what `<details>` is for.

Reply:
Hmm I suppose you're right, though should we add a `<details>` tag with a `<summary>` of "Previous funds" now? So that we don't have to add one later.

Reply:
sure, sounds fine

Reply:
Done.